src/components/Logo/Logo.jsx

```diff
@@ -2,10 +2,10 @@ import React from "react";
 
 import PropTypes from "prop-types";
 
-import { gettext } from "../utils/l10n.js";
+import { gettext } from "../../utils/l10n.js";
 
-import { ReactComponent as LogoSVG } from "../media/svg/logo.svg";
-import "../style/components/logo.scss";
+import { ReactComponent as LogoSVG } from "../../media/svg/logo.svg";
+import "../../style/components/logo.scss";
```

Comment:
@schalkneethling What do you think about moving the scss and image files into the component/[component name] folders as well?

Reply:
Eventually, the idea is to import these from the mdn-minimalist npm module so probably not something we should worry too much about. Images are interesting though.

I guess if we refer to an image in the Sass it probably makes sense for this to live with the Sass inside mdn-minimalist? For images that are not referenced in Sass, such as the logo SVG here, it makes complete sense to have those be alongside the relevant component.

Let me know your thoughts.

Reply:
That's a good point-- it's not something that we'll heavily interact with daily so less of a concern! 

For images referenced in Sass, I think the tricky part is if we have a generic image that's used in two or more Sass files, then where would it live? At my last place-- where we had Sass, images, and ts/x files living in the component folder-- I remember it felt a bit odd to reach into another component folder to grab an image. Maybe it's better to leave the images where they are for now?